server/src/metrics.rs

```diff
@@ -8,15 +10,76 @@ pub fn register_metrics(addr: std::net::SocketAddr) {
             0.0, 0.01, 0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9, 0.95, 0.99, 0.999,
         ])
         .expect("Metrics initialization failure");
-    // builder.add_allowed_address()
-    log::info!("Metrics http listener at {}", addr);
+
+    log::info!("Metrics http listener at {addr}");
     builder.install().expect("Metrics installation failure");
 
-    register_counter!("ya-relay.packets.incoming");
-    register_counter!("ya-relay.sessions.created");
-    register_counter!("ya-relay.sessions.purged");
-    register_counter!("ya-relay.sessions.removed");
+    register_counter!("ya-relay.packet.incoming.error");
```

Comment:
I'm concerned that copy-pasting these names will soon become error prone and hard-to-detect-failures-in-metrics. Can we gain more from a statically typed language?

Reply:
What do you suggest?